src/main/java/uk/gov/companieshouse/pscfiling/api/model/FilingDescription.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.pscfiling.api.model;
```

Comment:
I'd have thought this is more suited in package `uk.gov.companieshouse.pscfiling.api.config` as `FilingDataConfig`?

Reply:
moved and renamed as suggested